packages/api/lambdas/sf-starter.js

```diff
@@ -78,6 +78,11 @@ async function incrementAndDispatch(queueUrl, queueMessage) {
     return await dispatch(queueUrl, queueMessage);
   } catch (error) {
     await decrementQueueSemaphore(queueUrl);
+
+    if (error.code === 'ExecutionAlreadyExists' || error.message.includes('ExecutionAlreadyExists')) {
+      logger.info('Execution already exists. Proceeding to delete message from queue...');
+      await sqs.deleteSQSMessage(queueUrl, queueMessage.ReceiptHandle);
+    }
     throw error;
   }
```

Comment:
Opinions here would be appreciated. If we catch for this error code/message, and then delete the SQS message, line 86 will still throw the error so the CloudWatch logs will display something like:

```
Execution already exists. Proceeding to delete message from queue...
Error: ExecutionAlreadyExists
``` 

Will that be confusing? Should we throw a modified error?

Reply:
If the function returns successful, the message will be automatically deleted. Right?

Reply:
It's not automatically deleted, it's deleted here https://github.com/nasa/cumulus/blob/7472477f31cdedb9dbd522ebadab21b98044e302/packages/ingest/src/consumer.ts#L46 @jennyhliu